src/app/accounts/update-account-level.ts

```diff
@@ -0,0 +1,17 @@
+import { AccountsRepository } from "@services/mongoose"
+
+export const updateAccountLevel = async ({
+  id,
+  level,
+}: {
+  id: string
```

Comment:
We've been typing these at this point as `id: AccountId` to avoid the cast a few lines down as well